quicklens/spec.py

```diff
@@ -97,7 +97,7 @@ def __init__(self, tfname, lmax=None):
              * (optional) lmax  = maximum multipole to load (all multipoles in file will be loaded by default).
         """
         tarray = np.loadtxt(tfname)
-        lmin   = tarray[0, 0]
+        lmin   = int(tarray[0, 0])
         assert(lmin in [1,2])
```

Comment:
I think we still want this to fail if tarray[0,0] = 1.4, though with this change it would not. Would it work instead to

assert( int(lmin) == lmin )